pokesets/Manga/Aim to Be Battle King/PBK-Shin.yaml

```diff
@@ -0,0 +1,40 @@
+species: Empoleon
+setname: PBK-Shin
+ingamename: Empoleon
+displayname: Empoleon
+ability: [Torrent]
+item: [null]
+ball: [Poke]
+moves:
+- [Double Team]
+- [Aqua Jet]
+- [Rain Dance]
+- [Brine]
+gender: [m,f]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: null
```

Comment:
For these and others, pretty sure you can't have a null nature. The best substitute is to replace them with Hardy, Docile, Bashful, Quirky, or Serious. They make no changes to the Pokemon's stats.